CHANGELOG.asciidoc

```diff
@@ -32,6 +32,7 @@ endif::[]
 * Added support for setting service name and version for a transaction via the public api - {pull}2451[#2451]
 * Added support for en-/disabling each public annotation on each own - {pull}2472[#2472]
 * Added support for compressing spans - {pull}2477[#2477]
+* Added support for dropping fast exit spans - {pull}2491[#2491]
```

Comment:
Maybe this should be listed in `Potentially breaking changes`?

Reply:
We've decided to reverse the default on/off, so default should be 0ms, which means it's not a breaking change ... and sorry but you have a few changes needed to flip that for the config and the tests. We'll change the spec soon

Reply:
Ok, can I do the changes now or should I wait until the spec is officially changed?

Reply:
Done